php_phongo.c

```diff
@@ -979,15 +986,14 @@ void php_phongo_read_preference_to_zval(zval *retval, const mongoc_read_prefs_t
 #else
 		ADD_ASSOC_ZVAL_EX(retval, "tags", state.zchild);
 #endif
-	} else {
-		ADD_ASSOC_NULL_EX(retval, "tags");
 	}
 } /* }}} */
 
 void php_phongo_write_concern_to_zval(zval *retval, const mongoc_write_concern_t *write_concern) /* {{{ */
 {
 	const char *wtag = mongoc_write_concern_get_wtag(write_concern);
 	const int32_t w = mongoc_write_concern_get_w(write_concern);
+	const int32_t wtimeout = mongoc_write_concern_get_wtimeout(write_concern);
```

Comment:
I don't think that's a problem for initializers.... but a good point. But we do this in many other places too:

```
 858 php_phongo_server_description_type_t php_phongo_server_description_type(mongoc_server_description_t *sd)
 859 {
 860     const char* name = mongoc_server_description_type(sd);
 861     int i;
 862 
```
```
 872 void php_phongo_server_to_zval(zval *retval, mongoc_server_description_t *sd) /* {{{ */
 873 {
 874     mongoc_host_list_t *host = mongoc_server_description_host(sd);
 875     const bson_t       *is_master = mongoc_server_description_ismaster(sd);
 876     bson_iter_t         iter;
 877 
```

in the same file for example.